components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c

```diff
@@ -286,7 +296,11 @@ static lv_display_t *lvgl_port_add_disp_priv(const lvgl_port_display_cfg_t *disp
         buff_caps |= MALLOC_CAP_DEFAULT;
     }
 
-    /* Use RGB internal buffers for avoid tearing effect */
+#if CONFIG_IDF_TARGET_ESP32P4
```

Comment:
```suggestion
#if SOC_CACHE_INTERNAL_MEM_VIA_L1CACHE
```

Would this be better for forward compatibility?